xml/System.Windows.Data/BindingOperations.xml

```diff
@@ -190,6 +190,65 @@
         </remarks>
       </Docs>
     </Member>
+
+    <MemberGroup MemberName="EnableCollectionSynchronization">
```

Comment:
interesting: didn't know we had to manually add the overloads ourselves like that